css/style.css

```
@@ -15,5 +15,147 @@ body {
  * 
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
+ 
+ nav  {
+    display:flex;
+    justify-content: flex-end;
+     padding-top: 60px;
```

```suggestion
     padding-top: 3.75rem;
```
We should use relative units like rem instead of px where possible. This applies to the other px's in the rest of this file